changelog.md

```diff
@@ -1,5 +1,9 @@
 # Mango Program Change Log
 
+## v3.2.8
+Deployed: | Slot: 
+1. Add Change
```

Comment:
You typically update this only after the deployment, so it's ok to look like a placeholder for now?

Reply:
yeah